CODEOWNERS

```diff
@@ -1,3 +1,4 @@
 *                        @utilitywarehouse/pubsub
 /*/customer-billing/     @utilitywarehouse/org-billing-experience
 /*/finance/              @utilitywarehouse/finance-platform
+**/payment-platform/     @utilitywarehouse/payments-maintainers
```

Comment:
Actually we want to remain the owners for the msk settings, so, can you remove this please.